docs/resources/Channel.md

```diff
@@ -463,9 +463,14 @@ There are multiple message types that have a message_reference object.  Since me
 - These messages have `message_id` and `channel_id`, and `guild_id` if it is in a guild, with data of the message that was replied to. The channel_id and guild_id will be the same as the reply.
 - Replies are created by including a message_reference when sending a message. When sending, only `message_id` is required.
 
-###### Thread starter message
+###### Thread Created messages
 
-- These are the first message in a public thread.  They point back to the message in the parent channel from which the thread was started (type 21)
+- These are automatic messages sent when a public thread is created from an old message or without a message (type 18).
+- These messages have the `channel_id` and `guild_id` fields, with data of the created thread channel.
```

Comment:
optional: "with data of the" -> "which point to the"?
Because there's not really any extra data included here I think, right?

Reply:
the rest of them say "with data of the" so I just copied that; should I change it for all of them?

Reply:
That's so weird, oh well, I guess that's fine then